shell/platform/darwin/common/framework/Headers/FlutterChannels.h

```diff
@@ -143,9 +143,46 @@ FLUTTER_DARWIN_EXPORT
  * Adjusts the number of messages that will get buffered when sending messages to
  * channels that aren't fully set up yet.  For example, the engine isn't running
  * yet or the channel's message handler isn't set up on the Dart side yet.
+ *
+ * @param name The channel name.
+ * @param messenger The binary messenger.
+ * @param newSize The new size of the buffer.
+ */
++ (void)resizeChannelWithName:(NSString*)name
+              binaryMessenger:(NSObject<FlutterBinaryMessenger>*)messenger
+                      newSize:(NSInteger)newSize;
+
+/**
+ * Adjusts the number of messages that will get buffered when sending messages to
+ * channels that aren't fully set up yet.  For example, the engine isn't running
+ * yet or the channel's message handler isn't set up on the Dart side yet.
+ *
+ * @param newSize The new size of the buffer.
  */
 - (void)resizeChannelBuffer:(NSInteger)newSize;
 
+/**
+ * Toggles whether the channel should show warning messages when discarding messages
```

Comment:
"Toggle" means that it flips the state to the opposite, which isn't what this does; it sets it to a specific value. A toggle method wouldn't take a bool at all.

Reply:
I copied/pasted the comment from channel_buffers.dart
https://github.com/flutter/engine/blob/93e8901490e78c7ba7e319cce4470d9c6478c6dc/lib/ui/channel_buffers.dart#L562-L564
I will try to rephrase it.